firmware/integration/rusefi_config.txt

```diff
@@ -1373,8 +1370,12 @@ uint8_t[4] unusedsw;;"units", 1, 0, -20, 100, 0
 	uint16_t mc33_t_hold_off;;"us",   1,   0.0,        0,  10000,   0
 	uint16_t mc33_t_hold_tot;;"us",   1,   0.0,        0,  10000,   0
 
+	bit tcuEnabled
```

Comment:
please use one of existing unused bits, no reason to start a new bit array

Reply:
Where?

Reply:
Oh I found what you are talking about